Orchard1/Restart-Site/Restart-Site.psm1

```diff
@@ -28,22 +28,23 @@ function Restart-Site
         # If the path is invalid, then return an error.
         if (!(Test-Path ($Path)))
         {
-            Write-Error ("File or folder not found!")
+            Write-Error ('File or folder not found!')
             return
         }
 
         # Stopping IIS site and app pool.
-        Stop-IISSite $SiteName -confirm:$false
+        Stop-IISSite $SiteName -Confirm:$false
         Stop-WebAppPool $SiteName
 
         # Deleting bin and obj folders.
         # Add relative file paths here what you want to keep.
-        $whiteList = @("\src\Orchard.Azure\Orchard.Azure.CloudService\Orchard.Azure.WebContent\Bin\Startup\SetIdleTimeout.cmd")
-        # Also add the bin/obj folder's path of the paths in the whiteList here. This is needed for performance reasons, the script will run faster this way.
-        $whiteListFolders = @("\src\Orchard.Azure\Orchard.Azure.CloudService\Orchard.Azure.WebContent\Bin")
+        $whiteList = @('\src\Orchard.Azure\Orchard.Azure.CloudService\Orchard.Azure.WebContent\Bin\Startup\SetIdleTimeout.cmd')
+        # Also add the bin/obj folder's path of the paths in the whiteList here. This is needed for performance reasons,
+        # the script will run faster this way.
+        $whiteListFolders = @('\src\Orchard.Azure\Orchard.Azure.CloudService\Orchard.Azure.WebContent\Bin')
 
-        Get-ChildItem -Path ($Path + "\src\") -Recurse |
-            Where-Object { $PSItem.PSIsContainer -and ( $PSItem.Name -eq "bin" -or $PSItem.Name -eq "obj") } |
+        Get-ChildItem -Path ($Path + '\src\') -Recurse |
+            Where-Object { $PSItem.PSIsContainer -and ( $PSItem.Name -eq 'bin' -or $PSItem.Name -eq 'obj') } |
```

Comment:
Simplify with `-in`:
![image](https://user-images.githubusercontent.com/4007293/213821128-fa4483b7-2ee0-421b-b2fd-ba2dbbab5506.png)